junit5/src/main/java/cz/xtf/junit5/extensions/SkipForCondition.java

```diff
@@ -32,8 +31,10 @@ public ConditionEvaluationResult evaluateExecutionCondition(ExtensionContext con
 
 	public static ConditionEvaluationResult resolve(SkipFor skipFor) {
 		Image image = Image.resolve(skipFor.image());
-		Pattern name = Pattern.compile(skipFor.name());
-		if (name.matcher(image.getRepo()).matches()) {
+		DockerImageMetadata metadata = DockerImageMetadata.prepare(OpenShifts.master(), image);
```

Comment:
What does `DockerImageMetadata.prepare` do underneath? I hope it is not creating image stream.